src/components/ChatApp/MessageSection/MessageSection.react.js

```diff
@@ -368,11 +368,6 @@ class MessageSection extends Component {
     });
   };
 
-  addYouTube = newPlayer => {
-    this.pauseAllVideos();
-    this.setState(prevState => ({ player: [...prevState.player, newPlayer] }));
```

Comment:
Do not change the placing of function `addYouTube ` add the line `this.scrollToBottom();` here itself

Reply:
@fragm3 Done